src/hooks/useFormWithErrors.js

```diff
@@ -13,7 +13,8 @@ export const useFormWithErrors = (
 
   const [values, setValues] = useState(initialValues);
   const [errors, setErrors] = useState(initialFormErrors);
-  const [disabled, setDisabled] = useState(initialDisabled);
+  // const [disabled, setDisabled] = useState(initialDisabled);
+  let disabled = initialDisabled;
```

Comment:
we should change this back to what it previously was.

Check branch `main` for correct file

Reply:
fixed